docs/tutorials/dates.md

```diff
@@ -13,15 +13,15 @@ const userSchema = new mongoose.Schema({
 const User = mongoose.model('User', userSchema);
 ```
 
-When you create a user [document](/docs/documents.html), Mongoose will cast
+When you create a user [document](../../docs/documents.html), Mongoose will cast
```

Comment:
Ditto here, maybe just `../documents.html` instead of adding the extra `../docs`?